src/bin/ion/commands/beta/inspect.rs

```diff
@@ -113,29 +103,35 @@ pub fn run(_command_name: &str, matches: &ArgMatches<'static>) -> Result<()> {
         limit_bytes = usize::MAX
     }
 
-    let output: OutputRef;
+    let mut output: OutputRef;
```

Comment:
Now that we're not sharing the output stream via `Rc<RefCell<_>>`, we need to use explicit mutability.